docs/release-notes/1.5.5/release-1.5.5.md

```diff
@@ -0,0 +1,39 @@
+# [ML.NET](http://dot.net/ml) 1.5.5
+
+## **New Features**
+- **New API allowing confidence parameter to be a double**.([#5623](https://github.com/dotnet/machinelearning/pull/5623))
+. A new API has been added to accept double type for the confidence level. This helps when you need to have high precision then an int will allow for. (**Thank you @esso23**)
```

Comment:
did you mean `than` and not then?

Reply:
Yes I did. I always make that mistake, so thanks for catching it.